### PR TITLE
bugfix/presence-webhook-miss

### PR DESCRIPTION
**Pull request by baywet**
- fixes missing content on some pages

Note: there are a number of changes here that need to be reflected on v1 but I wanted to get feedback first. 
These changes include limitations for outlook & onedrive on GET/UPDATE/DELETE/POST pages, reordering and links in the path table in post subscription.

### CODE REVIEW

File: api-reference/beta/api/subscription-delete.md [hunk 1]
```diff
@@ -30,19 +34,36 @@ Depending on the resource and the permission type (delegated or application) req
...
 
-- In Outlook, delegated permission supports subscribing to items in folders in only the signed-in user's mailbox. That means, for example, you cannot use the delegated permission Calendars.Read to subscribe to events in another user’s mailbox.
+> **Note:** `/teams/allMessages` and `/chats/allMessages` are currently in preview. During the preview, you may use this API without fees, subject to the [Microsoft APIs Terms of Use](https://docs.microsoft.com/legal/microsoft-apis/terms-of-use?context=graph/context). However, users of apps that use the API might be required to have subscriptions to specific Microsoft 365 offerings. Upon general availability, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
+
+### driveItem (OneDrive)
+
+Additional limitations apply for subscriptions on OneDrive  items. The limitations apply to creating as well as managing (getting, updating, and deleting) subscriptions.
```
Comment (davidmu1) [inline api-reference/beta/api/subscription-delete.md:49]: extra space between OneDrive and items.

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md:49]:
```suggestion
Additional limitations apply for subscriptions on OneDrive items. The limitations apply to creating as well as managing (getting, updating, and deleting) subscriptions.
```

File: concepts/webhooks-with-resource-data.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ localization_priority: Priority
 
 # Set up change notifications that include resource data (preview)
 
-Microsoft Graph allows apps to subscribe to change notifications for resources via [webhooks](webhooks.md). You can now set up subscriptions to include the changed resource data (such as the content of a Microsoft Teams chat message) in change notifications. Your app can then run its business logic without having to make a separate API call to fetch the changed resource. As a result, the app performs better by making fewer API calls, which is beneficial in large scale scenarios.
+Microsoft Graph allows apps to subscribe to change notifications for resources via [webhooks](webhooks.md). You can now set up subscriptions to include the changed resource data (such as the content of a Microsoft Teams chat message or Microsoft Teams presence information) in change notifications. Your app can then run its business logic without having to make a separate API call to fetch the changed resource. As a result, the app performs better by making fewer API calls, which is beneficial in large scale scenarios.
```
Comment (davidmu1) [inline concepts/webhooks-with-resource-data.md:11]: Would remove the word now from "You can now set..." to just "You can set..."

Comment (baywet) [reply concepts/webhooks-with-resource-data.md:11]:
```suggestion
Microsoft Graph allows apps to subscribe to change notifications for resources via [webhooks](webhooks.md). You can set up subscriptions to include the changed resource data (such as the content of a Microsoft Teams chat message or Microsoft Teams presence information) in change notifications. Your app can then run its business logic without having to make a separate API call to fetch the changed resource. As a result, the app performs better by making fewer API calls, which is beneficial in large scale scenarios.
```

File: api-reference/beta/api/subscription-delete.md [hunk 3]
```diff
@@ -30,19 +34,36 @@ Depending on the resource and the permission type (delegated or application) req
 |[group conversation](../resources/conversation.md) | Group.Read.All | Not supported | Not supported |
 |[list](../resources/list.md) | Sites.ReadWrite.All | Not supported | Sites.ReadWrite.All |
 |[message](../resources/message.md) | Mail.ReadBasic, Mail.Read | Mail.ReadBasic, Mail.Read | Mail.ReadBasic, Mail.Read |
+|[presence](../resources/presence.md) | Presence.Read.All | Not supported | Not supported |
 |[security alert](../resources/alert.md) | SecurityEvents.ReadWrite.All | Not supported | SecurityEvents.ReadWrite.All |
 |[user](../resources/user.md) | User.Read.All | User.Read.All | User.Read.All |
 
-> **Note:** There are additional limitations for subscriptions on OneDrive and Outlook items. The limitations apply to creating as well as managing subscriptions (getting, updating, and deleting subscriptions).
+### chatMessage (Microsoft Teams)
```
Comment (Lauragra) [inline api-reference/beta/api/subscription-delete.md:41]: Is the content in the H3s in this topic directly related to Permissions? Or is there a better place for this information? I'm also unclear as to whether we should use the product names, or the workload name in the Graph docs - i.e. Teamwork.

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md:41]: I was simply reflecting the change from the POST page as it was missing on the other pages. And I wanted this information to be right where the API is described so people can't miss it.
That being said, we may be able to move that content that's on all the pages to an include to facilitate maintenance if that makes it better?

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md:41]: ping on the above @Lauragra

Comment (Lauragra) [reply api-reference/beta/api/subscription-delete.md:41]: Making an include seems like a good idea. I recommend removing the parenthetical product name references as well. Does the content belong under permissions?

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: done and done. Not necessarily but some of the content provided in there will related to permissions. See https://github.com/microsoftgraph/microsoft-graph-docs/pull/8943

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: @Lauragra ping on this one

Comment (Lauragra) [reply api-reference/beta/api/subscription-delete.md]: @baywet , it looks to me like the includes are working, but we're getting API Doctor errors. Do you have any ideas?

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: The tool seems to believe the includes are actually badly defined links for some reason. I'm not sure why.

Comment (baywet) [reply api-reference/beta/api/subscription-delete.md]: This has been stuck for too long, reverting the include change

**Comment by CelesteDG** [comment]
@baywet - Confirming that I'm merging this to master per 'ready to merge' tag. Thanks.

**Comment by baywet** [comment]
@CelesteDG yep, it's all good, thanks!

**Pull request merged**